src/Beman/Optional26/tests/optional_range_support.t.cpp

```diff
@@ -213,7 +180,7 @@ TEST(RangeSupportTest, LoopOverEmptyRange) {
     beman::optional::optional<int> empty;
     ASSERT_FALSE(empty.has_value());
 
-    for (auto _ : empty) {
+    for ([[maybe_unused]] auto _ : empty) {
```

Comment:
Does this make the compiler less unhappy about _ not being used?

Reply:
At least for some compilers :D